### ISSUE TITLE
[Suggestion] Flame Thrower

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO#3002]
This is a slight expansion on the heat post #2995 . I know that more in depth heat mechanics are something to come in the future, but here is an idea for a new weapon.

Possibly one of the best tactics for a fighter is to be able to fly under an enemies shield, but none of lasers, beams, and missiles (oh my) would be especially effective at this compared to firing at a distance. So an idea for a short range weapon is a space flamethrower (or plasma launcher, whatever). 

This weapon would be very short range of a few hundred meters max, have almost little effect on shields, but melt away future armor and damage hull. It can also include heat damage as well as the physical being that it is a flamethrower. Since it is short range it will be best used by agile fighters who can get close to enemy ships to burn them. Larger ship can still use them as anti-fighter, but in general it wouldn't be a mainline weapon since large ships would have trouble getting close to one another. And finally, not many games can say there are space flamethrowers. Would look great.

![](https://user-images.githubusercontent.com/36980446/94375289-854f7100-00c7-11eb-9f8a-d99382c1ebb0.gif)

**Comment by tsunamayo** [comment]
For when the fire mechanic is implemented ;)

**Comment by ZachZent** [comment]
I'm calling this a yes. Space flamethrower time!!!

**Comment by Womble-TC** [comment]
Once we get fire, can we get fire extinguishers? Could be a good way of putting out the fires, and also propelling yourself through a 0g environment...

**Comment by tsunamayo** [comment]
@Womble-TC yep, hopefully we would have crew use this and repair to fix your ship.

**Comment by MrSquid1** [comment]
These sound like a great counter to fighter style weapons, not much crew if any and hard to hit? Take an AOE weapon with fires that will require you to stop flying in order to repair
